cmd/podman/port.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"github.com/projectatomic/libpod/libpod"
+	"strings"
+	"strconv"
+)
+
+var (
+	portFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name: "all, a",
+			Usage: "display port information for all containers",
+		},
+		LatestFlag,
+	}
+	portDescription = `
+   podman port
+	
+	List port mappings for the CONTAINER, or lookup the public-facing port that is NAT-ed to the PRIVATE_PORT
+`
+
+	portCommand = cli.Command{
+		Name:        "port",
+		Usage:       "List port mappings or a specific mapping for the container",
+		Description: portDescription,
+		Flags:       portFlags,
+		Action:      portCmd,
+		ArgsUsage:   "CONTAINER-NAME [mapping]",
+	}
+)
+
+func portCmd(c *cli.Context) error {
+	var userProto, containerName string
+	var userPort int
+	var container *libpod.Container
+	var containers []*libpod.Container
+
+	args := c.Args()
+	if err := validateFlags(c, portFlags); err != nil {
+		return err
+	}
+
+	if c.Bool("latest") && c.Bool("all") {
```

Comment:
latest is not mentioned above as being supported.

Reply:
are you saying you dont see it as a flag ?

Reply:
Oh right, I forgot about your latestflag thing above.  Never mind.